sources/packages/backend/apps/queue-consumers/src/processors/schedulers/institution-integration/ier12-integration/_tests_/e2e/ier12-integration.scheduler.e2e-spec.ts

```diff
@@ -390,4 +390,147 @@ describe(describeProcessorRootTest(QueueNames.IER12Integration), () => {
     );
   });
+
+  it("Should generate an IER12 file with one record for a student with one sent disbursement that had a feedback error reported (DISE).", async () => {
+    // Arrange
+    const testInputData = {
+      student: JANE_MONONYMOUS_FROM_OTHER_COUNTRY,
+      application: {
+        applicationNumber: defaultApplicationNumber,
+        studentNumber: undefined,
+        relationshipStatus: RelationshipStatus.Single,
+        applicationStatus: ApplicationStatus.Completed,
+        applicationStatusUpdatedOn: undefined,
+      },
+      assessment: {
+        triggerType: AssessmentTriggerType.OriginalAssessment,
+        assessmentDate: undefined,
+        workflowData: WORKFLOW_DATA_SINGLE_INDEPENDENT_WITH_NO_DEPENDENTS,
+        assessmentData: ASSESSMENT_DATA_SINGLE_DEPENDENT,
+        disbursementSchedules: [
+          {
+            coeStatus: COEStatus.completed,
+            disbursementScheduleStatus: DisbursementScheduleStatus.Sent,
+            disbursementDate: undefined,
+            updatedAt: undefined,
+            dateSent: undefined,
+            disbursementValues: AWARDS_SINGLE_DISBURSEMENT_NO_BC_FUNDING,
+          },
+        ],
+      },
+      educationProgram: PROGRAM_GRADUATE_DIPLOMA_WITH_INSTITUTION_PROGRAM_CODE,
+      offering: OFFERING_FULL_TIME,
+    };
+    const application = await saveIER12TestInputData(db, testInputData, {
+      programYearPrefix: sharedProgramYearPrefix,
+      submittedDate: referenceSubmissionDate,
+    });
+    // Create a feedback error associated with the disbursement.
+    const [errorCode] = FULL_TIME_DISBURSEMENT_FEEDBACK_ERRORS;
+    const [disbursementSchedule] =
+      application.currentAssessment.disbursementSchedules;
+    const feedbackError = createDisbursementFeedbackError(
+      { disbursementSchedule },
+      { initialValues: { errorCode, updatedAt: dateUtils.addDays(-1) } },
+    );
+    await db.disbursementFeedbackErrors.save(feedbackError);
+
+    // Queued job.
+    // No date provided as it is expected that the disbursement feedback error
+    // date would ensure the IER12 record is generated.
+    const job = createIER12SchedulerJobMock();
```

Comment:
I believe the IER here is running based on current date -1 and picking the IER record based on feedback error table.
Amazing to see this level of validation in E2E ❤️